src/sdk/resources/policies.ts

```diff
@@ -29,8 +29,12 @@ export class PoliciesResource extends PagedResource<Policy> {
 		return getTypedResponse<EntityTypeAssignment>(result);
 	}
 
-	async deleteEntityAssignment(id: string, entityTypeId: string, token?: CancellationToken): Promise<boolean> {
-		return await this.httpClient.delete(`${this.resourcePath}/${id}/entity_types/${entityTypeId}`, null, token);
+	deleteEntityAssignment(id: string, entityTypeId: string, token?: CancellationToken): Promise<boolean> {
+		return this.httpClient.delete(`${this.resourcePath}/${id}/entity_types/${entityTypeId}`, null, token);
```

Comment:
Why did you remove await?

Reply:
Because we're not using the value of the promise here